src/ReleaseTests/ReleaseTest.class.st

```diff
@@ -410,6 +410,21 @@ ReleaseTest >> testRPackageOrganizer [
 		description: 'The default package organizer is the not the only instance of RPackageOrganizer'
 ]
 
+{ #category : #tests }
+ReleaseTest >> testRatioOfCommentedClasses [
+
+	| stereotypes violating |
+	stereotypes := OrderedCollection new.
+	stereotypes
+		addAll: Smalltalk allTraits;
+		addAll: Smalltalk allClasses.
+
+	stereotypes removeAll: TestCase allSubclasses.
+	stereotypes removeAll: BaselineOf allSubclasses.
+	violating := stereotypes select: [ :each | each comment isEmpty ].
+	self assert: violating size <= 914
```

Comment:
Maybe split the method into two, one that computes the violating classes and one that asserts its size

Reply:
Why?